accessibility/core/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
         targetSdkVersion 22
     }
 
-    compileSdkVersion 22
+    compileSdkVersion 23
     buildToolsVersion "22.0.1"
```

Comment:
Update these as well while you're at it (maybe target versions && support lib too, as they should go all together)